rmgpy/solver/base.pyx

```diff
@@ -53,6 +53,8 @@ import itertools
 from cpython cimport bool
 from rmgpy.quantity import Quantity
 from rmgpy.chemkin import getSpeciesIdentifier
+from rmgpy.reaction import Reaction
+from rmgpy.rmg.model import Species
```

Comment:
It seems you mainly use this for isinstance checks. Would it be better to check against the parent class `rmgpy.rmg.species.Species`? That would still return true for derived classes, and would prevent the need to change this when we combine the two Species classes.

Reply:
Ok, I take it you mean `rmgpy.species.Species`?

Reply:
Yup, that was a typo.